app.js

```diff
@@ -0,0 +1,29 @@
+const express = require ('express')
+const app = express()
+//app.use(express.json())
```

Comment:
This shouldn't be commented out, it should not even be there. When you are working with tickets, you should really only write code you KNOW you'll need, not code you *think* you might need later down the line